# Thread 1
ForLoopWithTryCatchFinallyRepresentationTest.operation() at :0
  escape = "START" at ForLoopWithTryCatchFinallyRepresentationTest.kt:26
  i = 1 at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
  loop(2 iterations) at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
    <iteration-1> at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
      a = 1 at ForLoopWithTryCatchFinallyRepresentationTest.kt:28
      escape = "try-1" at ForLoopWithTryCatchFinallyRepresentationTest.kt:33
      escape = "finally-1" at ForLoopWithTryCatchFinallyRepresentationTest.kt:37
      i = 2 at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
    <iteration-2> at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
      a = 2 at ForLoopWithTryCatchFinallyRepresentationTest.kt:28
  e = RuntimeException@1 at ForLoopWithTryCatchFinallyRepresentationTest.kt:34
  escape = "catch-2" at ForLoopWithTryCatchFinallyRepresentationTest.kt:35
  escape = "finally-2" at ForLoopWithTryCatchFinallyRepresentationTest.kt:37
  i = 3 at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
  loop(2 iterations) at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
    <iteration-1> at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
      a = 3 at ForLoopWithTryCatchFinallyRepresentationTest.kt:28
      escape = "try-3" at ForLoopWithTryCatchFinallyRepresentationTest.kt:33
      escape = "finally-3" at ForLoopWithTryCatchFinallyRepresentationTest.kt:37
      i = 4 at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
    <iteration-2> at ForLoopWithTryCatchFinallyRepresentationTest.kt:27
  escape = "END" at ForLoopWithTryCatchFinallyRepresentationTest.kt:40
